UpdateProfile/handler.ts

```diff
@@ -97,10 +99,35 @@ export function UpdateProfileHandler(
       profilePayload.email !== undefined &&
       profilePayload.email !== existingProfile.email;
 
+    // Check if servicePreferencesSettings mode changed
+    const servicePreferencesSettingsModeChanged =
+      profilePayload.service_preferences_settings !== undefined &&
+      profilePayload.service_preferences_settings.mode !==
+        existingProfile.servicePreferencesSettings.mode;
+
+    // check if user is sending LEGACY as mode
```

Comment:
Can we change this comment with a more clear description? for es.
```
// returns to LEGACY profile from updated ones is forbidden 
```

Reply:
Ok, seems legit.